src/Search/ElementalSolrIndex.php

```diff
@@ -1,25 +0,0 @@
-<?php
-
-namespace DNADesign\Elemental\Search;
-
-use Page;
-use SilverStripe\FullTextSearch\Solr\SolrIndex;
-
-if (!class_exists(SolrIndex::class)) {
-    return;
-}
-
-/**
- * Provides ability to index Elemental content for a page, so it can be returned in the context of the page
- * that the elements belong to
- */
-class ElementalSolrIndex extends SolrIndex
```

Comment:
You could also just deprecate this without deleting it here or changing the master alias

Reply:
Done in #711